README.md

```diff
@@ -1 +1,45 @@
-# cs224n_2020
\ No newline at end of file
+## __CS224n: Natural Language Processing with Deep Learning__
```

Comment:
Should be header 1

Reply:
Header 1 làm title xuống dòng trông hơi xấu @tiena2cva

Reply:
English, pls

The title should be header 1. If you want a beautiful title, you need to change the title to shorter